### ISSUE TITLE
Add whether each setting requires a reboot

### ISSUE THREAD

**Issue by dom123dom** [NetAppDocs/netapp-solutions#187]
Page: [Recommended ESXi host and other ONTAP settings](https://docs.netapp.com/us-en/netapp-solutions/virtualization/vsphere_ontap_recommended_esxi_host_and_other_ontap_settings.html)

Can we add a column which shows whether each of the settings requires the ESXi host to be rebooted or not?

For example these settings do require a reboot:
NFS.MaxQueueDepth
Net.TcpipHeapMax
Net.TcpipHeapSize

These do not:
NFS.HeartbeatTimeout
NFS.HeartbeatMaxFailures
NFS.HeartbeatFrequency

**Comment by kevin-hoke** [comment]
@ChrisReno - can we address the question so the issue can be closed?

**Comment by ChrisReno** [comment]
I have determined that the following settings require a reboot: 

Net.TcpipHeapSize
Net.TcpipHeapMax

All other settings do not. I went through the advanced settings of all the things that VSC changes to confirm. It would say "REQUIRES REBOOT" if it was affected. 

Reference: https://docs.netapp.com/us-en/vsc-vasa-provider-sra-97/deploy/reference-esx-host-values-set-by-vsc-for-vmware-vsphere.html#esxi-advanced-configuration

Example: 
<img width="855" alt="example_netTcpipHeapSize" src="https://user-images.githubusercontent.com/58820388/183139481-9f4cd90d-cc67-4844-8551-cd6890ebf634.png">
